packages/terra-site/src/examples/text/TextWeights.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Text from 'terra-text';
+
+const TextWeights = () => (
+  <div>
```

Comment:
And like with the Header, let's include an example portion of the page that has various mixtures of the props.